docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1451,8 +1451,7 @@ public class Customer {
     private Customer previousCustomer;
     @NextElementShadowVariable(sourceVariableName = "customers")
     private Customer nextCustomer;
-    @CascadingUpdateShadowVariable(targetMethodName = "updateArrivalTime", sourceVariableName = "vehicle")
-    @CascadingUpdateShadowVariable(targetMethodName = "updateArrivalTime", sourceVariableName = "previousCustomer")
+    @CascadingUpdateShadowVariable(targetMethodName = "updateArrivalTime", sourceVariableNames = {"vehicle", "previousCustomer"})
```

Comment:
Now would be the time to add a section of the documentation that shows how piggybacks interact with cascading variables.